docs/concepts/architecture.md

```diff
@@ -113,3 +113,43 @@ Nuclio provides the ability to run different versions of the same function simul
 
 The event-source mapping can either specify the exact function version, or utilize a defined alias and thus eliminate the need to change the mapping when a newer version of the function is published.
 
+
+## Processors' internal abstractions
```

Comment:
add a bullet at the top of the page for this section

Reply:
@jillnogold changed this chapter level to ####, so I believe not relevant anymore